src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,125 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private int size;
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
```

Comment:
Using raw types in Java generics is not safe because it can lead to ClassCastException at runtime. You should use the diamond operator to ensure type safety. Change `new Node[DEFAULT_CAPACITY]` to `new Node<K,V>[DEFAULT_CAPACITY]`. However, Java does not allow the creation of generic arrays, so you'll need to create an array of raw types and cast it: `(Node<K, V>[]) new Node<?,?>[DEFAULT_CAPACITY]`.